client/src/components/ResponsiveLayout.js

```diff
@@ -118,7 +118,12 @@ class ResponsiveLayout extends Component {
 						}}
 					/>
 					{(pageProps.useNavigation === true || floatingMenu === true) &&
-						<div style={sidebarContainer} className={ floatingMenu === false ? "hidden-xs" : "nav-overlay"}>
+						<div
+							style={sidebarContainer}
+							className={floatingMenu === false ? "hidden-xs" : "nav-overlay"}
+							name="mainSidebar"
```

Comment:
`name` attribute is not needed.

Reply:
Wanted to mirror mainViewport code for consistency. Should this be adjusted in both divs?

Reply:
There's no styling for `#main-viewport` so the id attribute could simply be removed (as could the name attribute).
[Strictly speaking, these changes do not pertain to the issue at hand, so at least make a separate commit with a clear message.]

Reply:
#main-viewport is used in the scrollLinkNavItem, I will let it be for now.
I recall that the scroll-spy might also look at the name, I'm not quite sure, but there was something that might require the name property

Reply:
OK, but back to the original comment: _that_ div doesn't need a name.